apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -63,21 +63,38 @@ This project comes with TypeScript configured and has the following structure.
 
 ### Configuring the project to use PostgreSQL
 
+Prisma init migrations will try to drop `postgres` database by default, and this may lead to conflicts with Supabase initial migrations that are critically needed for the correct functioning of our managed databases. For such cases Prisma has a feature called [Shadow Databases](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).
+
+Create a shadow database with the local database (or create another database in your PostgreSQL server with the same Supabase project) using the `psql` CLI and the `DATABASE_URL` from the previous steps.
```

Comment:
```suggestion
Create a shadow database in your PostgreSQL server within the same Supabase project using the `psql` CLI and the `DATABASE_URL` from the previous steps (or use the local database).
```

Reply:
Thanks for the fixes!
I think the first sentence is quite good now.
Wdyt about the second one? I have a feeling that it would be more correct, but not sure about my wording

Reply:
LGTM!